core/watch

```diff
@@ -29,7 +29,7 @@ require_once __DIR__ . '/../vendor/autoload.php';
         };
 
         $watcher = new ResourceWatcher($tracker);
-        $watcher->track('source', __DIR__ . '/../source');
+        $watcher->track('source', __DIR__ . '/../build/source');
```

Comment:
Could get overwritten on a PatternLab update; as long as that's understood at time of the update process.